### ISSUE TITLE
VM keeps auto stopping even after disabling or deleting AutoStop logic app

### ISSUE THREAD

**Issue by KerryRJ** [microsoft/startstopv2-deployments#54]
Any ideas on how to troubleshoot this?

**Comment by alurbina** [comment]
Hello,

Can we verify if the VM has any associated alerts? If so, can we delete / disable them? If this does not work, can you share with us more details of the VM?

**Comment by KerryRJ** [comment]
The VM is a Standard D2ds v5 (2 vcpus, 8 GiB memory) created via AVD.

> verify if the VM has any associated alerts

Where do I find these VM associated alerts?

I eventually deleted and recreated the VMs automation group, and haven't yet configured auto stop. Knowing where and what to look for reguarding the associated alerts would be handy.

**Comment by balas77** [comment]
To verify the VM alerts:  Navigate to your VM in the Azure portal --> click Alerts (under Monitoring tab) --> click Alert Rules. Make sure you select the filter as enabled. Once you find the alert, then either you can disable or delete it.

![image](https://user-images.githubusercontent.com/64924532/165860866-4d5a47b8-d8b1-4cd1-8793-41ab8209100f.png)

![image](https://user-images.githubusercontent.com/64924532/165860968-dc0fdd23-12d3-477c-b69c-03278b331c5d.png)

![image](https://user-images.githubusercontent.com/64924532/165861372-83bbdef3-0609-4f6a-adf4-1f2e61658b69.png)

**Comment by KerryRJ** [comment]
That was really helpful. The auto stop alert was still there and enabled after I had deleted the start stop resource group. It is also did not seem to have updated or disabled even though I had tried disabling the auto stop logic app. Is this by design?

**Comment by balas77** [comment]
Yes by design the alert will be created on your VMs which is outside of the Start/Stop resource group so even if you delete the Start/Stop RG, this will **not** going to delete the alert which is part of your VM resource. 

Also disabling/deleting the logic app is not going to take any effect to delete the alert as the logic app is just an orchestrator to call the function app which is responsible to create the alert based on the payload you have configured.  So if you want to disable the alert then you can add the VM in the payload exclude list (refer our [doc](https://docs.microsoft.com/en-us/azure/azure-functions/start-stop-vms/deploy#scheduled-start-and-stop-scenario) here for sample exclude list) in the logic app.

**Comment by KerryRJ** [comment]
If configuration in the Logic App is modified, are any settings there carried over to the alert rule?

**Comment by balas77** [comment]
Yes new alert will be created on the VM (old alert will be deleted) whenever the logic app runs so if you change the value of any property (say AutoStop_Threshold : 10) in the auto stop payload then new alert will be created based on the updated value.
